show/main.py

```diff
@@ -228,6 +228,35 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'Neighbor MetaData' subcommand ("show interfaces neighbor")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+def neighbor(interfacename):
+    """Show Interface Neighbor"""
+    neighborCmd = 'sonic-cfggen -d --var-json "DEVICE_NEIGHBOR"'
+    p1 = subprocess.Popen(neighborCmd, shell=True, stdout=subprocess.PIPE)
+    neighbor_dict = json.loads(p1.stdout.read())
+
+    neighborMetaDataCmd = 'sonic-cfggen -d --var-json "DEVICE_NEIGHBOR_METADATA"'
+    p2 = subprocess.Popen(neighborMetaDataCmd, shell=True, stdout=subprocess.PIPE)
+    neighborMetaData_dict = json.loads(p2.stdout.read())
+
+    #Swap Key and Value
+    tempDict= {}
+    for port in natsorted(neighbor_dict.keys()):
+        tempDict[neighbor_dict[port]['name']]= {'localPort':port, 'neighborPort':neighbor_dict[port]['port']}
```

Comment:
Please add space before `=` and after colons,

`tempDict[neighbor_dict[port]['name']] = {'localPort': port, 'neighborPort': neighbor_dict[port]['port']}`